spot-client/src/spot-tv/calendars/calendarService.js

```diff
@@ -34,6 +34,8 @@ export class CalendarService extends EventEmitter {
          * A cache of the last calendar events that have been fetched.
          */
         this._calendarEvents = [];
+
+        this._hasFetchedEvents = false;
```

Comment:
hmm did you mean `_hasUpdatedEvents` or to use `_hasFetchedEvents` in the code below ? the var is named differently in all 3 places

Reply:
I'll spend the rest of the day trying to forget this ever happened.